hazelcast/src/main/java/com/hazelcast/map/LocalMapStats.java

```diff
@@ -274,4 +274,6 @@
      */
     Map<String, LocalIndexStats> getIndexStats();
 
+    boolean isNativeMemoryUsed();
```

Comment:
We need javadoc here. When we add this method here, it also becomes public API, not only used by MC.